Sources/IssueReporting/TestContext.swift

```diff
@@ -1,8 +1,8 @@
-/// A type representing the context in which a test is being run, i.e. either in Swift's native
+/// A type representing the context in which a test is being run, _i.e._ either in Swift's native
 /// Testing framework, or Xcode's XCTest framework.
 public enum TestContext {
   /// The Swift Testing framework.
-  case swiftTesting
+  case swiftTesting(Testing)
```

Comment:
Technically this is a breaking change for folks referring directly to this `.swiftTesting` case, so we could introduce a new set of alternate APIs here instead.

If someone was merely switching over the cases in their test helper it should be source compatible, though.